src/components/pages/datasets/person/utils.js

```diff
@@ -0,0 +1,6 @@
+import { SOURCE } from './const';
+
+export const checkSource = (data) => {
+  const source = Object.values(SOURCE).find((item) => new RegExp(item.title).test(data.source));
+  return source.translation;
+};
```

Comment:
1. throw Error if source not found.
2. Why RegExp?

Reply:
done